rudder-core/src/main/scala/com/normation/rudder/domain/reports/ReportType.scala

```diff
@@ -104,13 +104,17 @@ object ReportType {
         case (_, BadPolicyMode)       | (BadPolicyMode, _)        => BadPolicyMode
```

Comment:
You will have to change the start case above to AuditNotApplicable, else we will always get EnforceNotApplicable for them